### PR TITLE
🧪 Add initial comments for resistor-color-duo

### PR DESCRIPTION
**Pull request by SleeplessByte**
🧪 **experiment** 🧪

@iHiD

### CODE REVIEW

File: automated-comments/javascript/resistor-color-duo/commentary/must_optimise_helper.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+⚡ The helper method `%{method.name}` is not optimal. The helper can
+probably be the same as the solution to `resistor-color`. Mentor the student
+to retrieve their solution and/or optimise their helper.
```
Comment (iHiD) [inline automated-comments/javascript/resistor-color-duo/commentary/must_optimise_helper.md]:
```suggestion
to retrieve their solution and/or optimise their helper.

⚡ **Warning:** We cannot guarantee the analyzer worked correctly on this point. Please double-check if it made a mistake, and if it did please open an issue [here](https://github.com/exercism/javascript-analyzer/issues/new?assignees=&labels=%3Abug%3A+bug&template=incorrect-analysis.md&title=Incorrect+Analysis%3A+) so the analyzer can be updated.
```

File: automated-comments/javascript/resistor-color-duo/commentary/must_optimise_helper.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+⚡ The helper method `%{method.name}` is not optimal. The helper can
```
Comment (SleeplessByte) [inline automated-comments/javascript/resistor-color-duo/commentary/must_optimise_helper.md]:
```suggestion
The helper method `%{method.name}` is not optimal. The helper can
```

File: automated-comments/javascript/resistor-color-duo/commentary/must_declare_function.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+⚡ Ensure the method `%{method/name}` exists. It was not found when
```
Comment (SleeplessByte) [inline automated-comments/javascript/resistor-color-duo/commentary/must_declare_function.md]:
```suggestion
Ensure the method `%{method/name}` exists. It was not found when
```

File: automated-comments/javascript/resistor-color-duo/commentary/must_declare_function.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+analysing this solution. If it does not exist, point this out to the student.
```
Comment (SleeplessByte) [inline automated-comments/javascript/resistor-color-duo/commentary/must_declare_function.md]:
```suggestion
analysing this solution. If it does not exist, point this out to the student.

⚡ **Warning:** We cannot guarantee the analyzer worked correctly on this point. Please double-check if it made a mistake, and if it did please open an issue [here](https://github.com/exercism/javascript-analyzer/issues/new?assignees=&labels=%3Abug%3A+bug&template=incorrect-analysis.md&title=Incorrect+Analysis%3A+) so the analyzer can be updated.
```

File: automated-comments/javascript/resistor-color-duo/must_use_a_helper.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+Using a helper method is good practice, because it replaces a cryptic "member
+call" with a named call, that can be documented individually.
```
Comment (iHiD) [inline automated-comments/javascript/resistor-color-duo/must_use_a_helper.md]:
```suggestion
call" with a named call that can be documented individually.
```

In English, if there's a comma, use "which". If there's not a comma, use "that". Use "which" when the sentence would still make sense without the clause that follows it, and use that if it wouldn't.

So here, as "can be documented individually" is part of the reason, and things don't make sense without it, I'd use "that" and therefore no comma.

**Pull request merged**